pages/v/[id]/embed.tsx

```diff
@@ -0,0 +1,188 @@
+import { OPEN_SOURCE_URL, MUX_HOME_PAGE_URL, HOST_URL } from '../../../constants';
+import { HTMLVideoElementWithPlyr } from '../../../types';
+import logger from '../../../lib/logger';
+
+type Params = {
+  id: string;
+}
+
+export type Props = {
+  playbackId: string,
+  shareUrl: string,
+  poster: string
+};
+
+export const getStaticProps: GetStaticProps = async (context)  => {
+  const { params } = context;
+  const { id: playbackId } = (params as Params);
+  const poster = `https://image.mux.com/${playbackId}/thumbnail.png`;
+  const shareUrl = `${HOST_URL}/v/${playbackId}`;
+
+  return { props: { playbackId, shareUrl, poster } };
+};
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  return {
+    paths: [],
+    fallback: true,
+  };
+};
+
+
+interface AsteriskButtonProps {
+  onOpenOverlay?: () => void;
+}
+
+const AsteriskButton: React.FC<AsteriskButtonProps> = ({ onOpenOverlay }) => {
+  return (
+    <>
+      <a onClick={onOpenOverlay} onKeyPress={onOpenOverlay} role='button' tabIndex={0}>
+        <Asterisk size={25} />
+      </a>
+      <style jsx>{`
+        a {
+          background: var(--plyr-video-control-background-hover,var(--plyr-color-main,var(--plyr-color-main,#00b3ff)));
+          border-radius: 50%;
+          padding: 5px;
+          display: flex;
+          cursor: pointer;
+        }
+
+        a:hover {
+          opacity: 0.5;
+        }
+      `}</style>
+    </>
+  );
+};
+
+const PlaybackEmbedded: React.FC<Props> = ({ playbackId, poster }) => {
+  const videoRef = useRef<HTMLVideoElementWithPlyr>(null);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [showOverlay, setShowOverlay] = useState(false);
+  const [wasPlaying, setWasPlaying] = useState(false);
+  const router = useRouter();
+
+  const onCloseOverlay = () => {
+    setShowOverlay(false);
+
+    if (wasPlaying) {
+      videoRef.current?.play();
+    }
+  };
+
+  const onOpenOverlay = () => {
+    setWasPlaying(videoRef.current?.plyr.playing || false);
+
+    videoRef.current?.pause();
+
+    setShowOverlay(true);
+  };
+
+  const nonEmbedUrl = `/v/${playbackId}`;
+
+  if (router.isFallback) {
+    return (
+      <Layout
+        metaTitle="View this video created on stream.new"
+        image={poster}
+        centered
+        darkMode
+      >
+        <FullpageLoader text="Loading player..." />;
+      </Layout>
+    );
+  }
+
+  const onError = (evt: ErrorEvent) => {
+    setErrorMessage('This video does not exist');
+    setIsLoaded(false);
+    logger.error('Error', evt);
+  };
+
+  const showLoading = (!isLoaded && !errorMessage);
+
+  return (
+    <>
+      {errorMessage && <h1 className="error-message">{errorMessage}</h1>}
+      {showLoading && <FullpageLoader text="Loading player" />}
+      <div className="wrapper">
+        <VideoPlayer ref={videoRef} playbackId={playbackId} poster={poster} onLoaded={() => setIsLoaded(true)} onError={onError} />
```

Comment:
I just realized, when using the main video player vs. embed video player, we should reflect the difference in the Mux Data integration. Pass in a different value that can be used for `player_name` metadata